yarn.lock

```diff
@@ -2662,14 +2680,14 @@
     flat "^5.0.2"
     react-animate-height "^3.0.4"
 
-"@opentripplanner/trip-form@5.0.0":
-  version "5.0.0"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/trip-form/-/trip-form-5.0.0.tgz#80c676107ea52791efe28a4e7ec647aec61ca793"
-  integrity sha512-8y6HdlRbgu91Xw8FO2sAgmtkPl1o2DhhxHDK9FxTiXUPOBFdj5pVYSG2v5qQFtO6bIy6jWFP7e3rdD0R3rxniQ==
+"@opentripplanner/trip-form@6.0.0":
+  version "6.0.0"
+  resolved "https://registry.yarnpkg.com/@opentripplanner/trip-form/-/trip-form-6.0.0.tgz#1c0c09ea675b8254faf20f656f68252bc291f168"
+  integrity sha512-LrgVx6HmABot0hB8owlYvvzG2vO4eEdGFiiQR3Cs2zeUxz22qArAJoTDSLRuLhys0xYlLv8VB+J3ICWJaY7UnA==
   dependencies:
     "@floating-ui/react" "^0.19.2"
```

Comment:
Curious: do we still need this dependency? I don't think it is being used anymore.

Reply:
Sorry, revert this change to yarn.lock. I was just pointing it out. Dependency removal needs to happen with the corresponding OTP-UI package.